### PR TITLE
Swift 4 Support

### PR DESCRIPTION
**Pull request by pmairoldi**
Migrates code to swift 4.

### CODE REVIEW

File: ChartsRealm.podspec [hunk 1]
```diff
@@ -10,6 +10,6 @@ Pod::Spec.new do |s|
   s.osx.deployment_target = "10.11"
   s.source = { :git => "https://github.com/danielgindi/ChartsRealm.git", :tag => "v#{s.version}" }
   s.source_files  = "ChartsRealm/Classes/**/*.swift"
-  s.dependency "Charts", "3.0.3"
+  s.dependency "Charts", "3.0.4"
   s.dependency "RealmSwift", "~> 2.10"
```
Comment (liuxuan30) [inline ChartsRealm.podspec:14]: shall we bump to 2.10.2 as well though it should be fine?

Comment (pmairoldi) [reply ChartsRealm.podspec:14]: No. We should leave it. All versions of 2.10 should be API compatible.

Comment (liuxuan30) [reply ChartsRealm.podspec:14]: sure. good to go then

Comment (pmairoldi) [reply ChartsRealm.podspec:14]: making sure everything still builds and the pod validates then ill merge

Comment (liuxuan30) [reply ChartsRealm.podspec:14]: sure. good to go then. My carthage update always timeout after a long time downloading realm swift :(

Comment (pmairoldi) [reply ChartsRealm.podspec:14]: You should use the build command when testing. It's `carthage build --no-skip-current`. Just builds stuff from checkout.

**Pull request merged**